### DISCUSSION TITLE
Terminal reads Japanese characters instead of language symbols

### DISCUSSION THREAD

**Discussion by theghostmac** [starship/starship#5160] [Q&A] [answered]
Hi please,

So I started using `starship` recently. I wanted to use the Nerd Font but the language icons are in Japanese. How do I fix this?
![image](https://user-images.githubusercontent.com/83423188/235578898-eb52bcf7-c5ab-49ba-9975-df3c0fa65e6c.png)

I'm almost convinced it's a system setting, not `starship` per say, but someone might know how to go about the fix here.

Thanks!

**Comment by mickimnet** [comment]
Are you using a _Starship_ preset, and if those, which? Furthermore, which _Nerd Font_ are you using?

**Comment by theghostmac** [comment, marked as answer]
Hi Mick,

Yes, I use a Preset named Nerd Font Symbols ([here](https://starship.rs/presets/nerd-font.html)). 
The reason it didn't work was because I didn't have Nerd Font pre-installed, my bad.

After installing Nerd Font, most of the symbols worked.  

Thank you.
